lib/core-integration/src/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -740,6 +740,148 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
   -- minting
   -- update with sign / submit tx where applicable
   -- end to end join pool and get rewards
+
+    it "TRANS_NEW_BALANCE_01a - multiple-output transaction with all covering inputs present" $ \ctx -> runResourceT $ do
+
+        liftIO $ pendingWith "No errMsg403transactionAlreadyBalanced is returned - to be fixed in ADP-656"
+
+        -- constructing source wallet
+        let initialAmt = minUTxOValue (_mainEra ctx)
+        wa <- fixtureWalletWith @n ctx [initialAmt]
+
+        -- the tx involes two outputs :
+        -- 999978
+        -- 999978
+        -- results in two changes
+        -- 49998927722
+        -- 49998927722
+        -- incurs the fee of
+        -- 144600
+        -- and involves one input
+        -- 100000000000
+        let serializedTxHex =
+                "84a600818258200eaa33be8780935ca5a7c1e628a2d54402446f96236c\
+                \a8f1770e07fa22ba8648000d80018482583901a65f0e7aea387adbc109\
+                \123a571cfd8d0d139739d359caaf966aa5b9a062de6ec013404d4f9909\
+                \877d452fc57dfe4f8b67f94e0ea1e8a0ba1a000f422a82583901ac9a56\
+                \280ec283eb7e12146726bfe68dcd69c7a85123ce2f7a10e7afa062de6e\
+                \c013404d4f9909877d452fc57dfe4f8b67f94e0ea1e8a0ba1a000f422a\
+                \825839011a2f2f103b895dbe7388acc9cc10f90dc4ada53f46c841d2ac\
+                \44630789fc61d21ddfcbd4d43652bf05c40c346fa794871423b65052d7\
+                \614c1b0000000ba42b176a82583901c59701fee28ad31559870ecd6ea9\
+                \2b143b1ce1b68ccb62f8e8437b3089fc61d21ddfcbd4d43652bf05c40c\
+                \346fa794871423b65052d7614c1b0000000ba42b176a021a000234d803\
+                \198ceb0e80a0f5f6" :: Text
+        let balancePayload = Json [json|{
+              "transaction": { "cborHex" : #{serializedTxHex}, "description": "", "type": "Tx AlonzoEra" },
```

Comment:
The type is always `"type": "Tx AlonzoEra"` it seems which also is also the only allowed according to API doc (https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/balanceTransaction). As far as I see (testing manually) there does not seem to be any checking of that value and you can basically put anything there...

Reply:
we don't use this information in api machinery as everything (information about compatibility with Alonzo Era or other era) is inside cborHex and we get this from deserialisation. The schema was imposed on us - please remember. Maybe we will validate the description, but now now.